### PR TITLE
[Docs] Translate docs/book/zh/src/kubernetes/demo.md to zh (#1106)

### PR DESCRIPTION
**Pull request by kgpp34**
this commit has translated docs/book/zh/src/kubernetes/demo.md to zh, please check for accuracy, thanks. If the translation have any questions, you can contact me directly by e-mail, the e-mail address is 974840768@qq.com

### CODE REVIEW

File: docs/book/zh/src/kubernetes/demo.md [hunk 1]
```diff
@@ -1 +1,16 @@
-# Demo apps
+# Demo 应用
+
+在本章中，我们将介绍两个 demo 应用。 我们将从 Rust 源码进行构建，并围绕它们构建 OCI 镜像，然后将这些镜像发布到 Docker Hub。
+
+如果您还没有这样做，请先
```
Comment (alabulei1) [inline docs/book/zh/src/kubernetes/demo.md:5]: you 翻译成 你

File: docs/book/zh/src/kubernetes/demo.md [hunk 2]
```diff
@@ -1 +1,16 @@
...
+
+* [安装 Rust](https://www.rust-lang.org/tools/install)
+* [注册 Docker Hub](https://hub.docker.com/)
+
+接下来，探索示例
+
+* [一个简单的 WASI 示例](demo/wasi.md)
+* [一个 HTTP 服务器示例](demo/server.md)
+
+由于我们已经在 Docker Hub 上构建并发布了这些 demo 应用，您也可以直接进入容器运行时部分来使用这些镜像。
```
Comment (alabulei1) [inline docs/book/zh/src/kubernetes/demo.md:15]: you 翻译成你

**Review by alabulei1 — COMMENTED** [review]
@kgpp34  Thanks for your contribution! Please check out my comments.

File: docs/book/zh/src/kubernetes/demo.md (hunk 2 above)
Comment (kgpp34) [reply docs/book/zh/src/kubernetes/demo.md]: already fixed

**Comment by alabulei1** [comment]
LGTM!

@kgpp34 thanks for your contribution!

@hydai  could you help merge this PR?

**Review by hydai — APPROVED** [review]

**Pull request merged**